microsoft/botframework-solutions
# Added extension methods for spoken versions of dates and ILists and updated the CreateEventDialog

## Description
Created a ToSpeechString extension method of IList<T> that allows  your to enumerate items in a list for use in the speak property of a response.
Created a ToSpeechDateString() and a ToSpeechTimeString() extension method for DateTime to create a string that can be used in a spoken response with dates and times.
Added unit tests for the methods.
Updated the CreateEventDialog to use these methods.
Updated some response strings in CreateEventDialog to make it more fluent.
Fixed several stylecop violations in CreateEventDialog.

## Testing Steps
Create a meeting using speech from the emulator and check the spoken responses (especially the create meeting summary).
